vllm/v1/core/single_type_kv_cache_manager.py

```diff
@@ -597,7 +612,10 @@ def remove_skipped_blocks(self, request_id: str, num_computed_tokens: int) -> No
         pass
 
     def get_num_common_prefix_blocks(
-        self, request_id: str, num_running_requests: int
+        self,
+        running_request_id: str,
+        running_request_ids: list[str],
```

Comment:
```suggestion
        num_running_requests: int
```
Do we only need this?

Reply:
@heheda12345 I made this small update to the logic now, You can review it.